src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -0,0 +1,269 @@
+using CalculatorUITestFramework;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace CalculatorUITests
+{ 
+    [TestClass]
+    public class StandardModeFunctionalTests
+    {
+        private StandardOperatorsPanel standardOperators => StandardCalculatorPage.StandardOperators;
+
+        /// <summary>
+        /// Initializes the WinAppDriver web driver session.
+        /// </summary>
+        /// <param name="context"></param>
+        [ClassInitialize]
```

Comment:
>[ClassInitialize] [](start = 8, length = 17)

Should we put this in Assembly Initialize instead?  This will result in us using the same calculator instance across classes, but I think that's ok.

Reply:
I think we could consider that once we have more classes if we find they all use the same ClassInitialize, but for now, I think it's ok to leave it here.